### ISSUE TITLE
[Accessibility] Inconsistent display order of TabPage pane in the UIA tree

### ISSUE THREAD

**Issue by MelonWang1** [dotnet/winforms#10365]
### .NET version

.NET 9.0.100-alpha.1.23564.26

### Did it work in .NET Framework?

Yes

### Did it work in any of the earlier releases of .NET Core or .NET 5+?

Yes, this issue cannot repro in .NET 3.1/5.0, can repro in .NET 6.0/7.0/8.0.

### Issue description

Actual result:
The TabPage1 pane is at the bottom of more/less buttons.
![core9 0](https://github.com/dotnet/winforms/assets/94418985/7ec4e97a-756c-43c5-aec6-752376e5292a)

Expected result:
The TabPage1 pane should be on the top of the more/less buttons.
![framework](https://github.com/dotnet/winforms/assets/94418985/6cde8f7c-7643-404e-921e-fd3602ae1d7e)

### Steps to reproduce

1. Create a Winforms .NET Core application
2. Add a TabControl
3. Add some tab to see more/less buttons
4. Build and run
5. Open Inspect or Accessibility Insights tool
6. Focus on TabPage1 pane

**Comment by Epica3055** [comment]
@lonitra @Tanya-Solyanik I need some help.

When I tried to change winfrom main branch to 5.0, since the issue says that 5.0 works fine, and I tried to execute Restore.cmd , it showed some errors.
So how can I run 5.0?

```
E:\winforms>Restore.cmd
Processing E:\winforms\global.json
cmake.exe already exists; replacing...
dotnet-api-docs_net5.0 was not found in .
Native tools bootstrap failed
MSBUILD : error MSB1001: Unknown switch.
Switch: -NativeToolsOnMachine

For switch syntax, type "MSBuild -help"
Build failed.
```

**Comment by Tanya-Solyanik** [comment]
@MelonWang1  - could you please share where did you install NET5 from?

**Comment by MelonWang1** [comment]
@Tanya-Solyanik  I'm using .NET 5.0(5.0.17) that comes with VS.

**Comment by Tanya-Solyanik** [comment]
FYI - https://github.com/dotnet/winforms/issues/4120 might or might not have a similar root cause

**Comment by Epica3055** [comment]
@Tanya-Solyanik thank you for the lead.

**Comment by JeremyKuhne** [comment]
Fallout from introducing our own UIA provider. No known way to solve, but there are workarounds for test. (Per @Tanya-Solyanik)

**Comment by merriemcgaw** [comment]
@Tanya-Solyanik can we close this issue? I don't think it's something that is impacting anything any longer.

**Comment by LeafShi1** [comment]
@merriemcgaw I think we can close this issue. As Jeremy said, the order problem of Spinner is not a problem of AccessibilityObject, but the way UIA handles system controls. Even if we don’t create our TabControlAccessibleObject, Spinner will appear directly in the UIA tree and be at the front. What do you think?
